template

```diff
@@ -133,4 +133,20 @@ Status {
     Font = "$FONT"
     Texture = "Solid $COLOR_STATUS_BG"
     Text = "$COLOR_STATUS_TEXT"
-}
\ No newline at end of file
+}
+
+Panel {
+	Background = "Solid $COLOR_PANEL"
+	Focused {
+		Font = "$FONT#Center"
+		Color = "#000000"
```

Comment:
Now the panel so just happens to have the same color in normal and dark mode, but wouldn't it be nice to have variables for these as well while at it?

Reply:
Yes, I added variables for the font colors.
I made the panel white because the focused menu background is also white, for both the normal and the dark theme variant.